types.tsx

```diff
@@ -6,7 +6,8 @@ export type RootStackParamList = {
 export type EpisodeInputStackParamList = {
   EpisodeInput: undefined;
   EpisodeDisplay: undefined;
-  CameraFlow: undefined;
+  EpisodeConfirmation: undefined;
+  EpisodeEdit: undefined;
 };
```

Comment:
this is a typescript thing: but what if this was an enum? Or its kinda just weird how there are a bunch of undefined. Maybe this is a problem for later?

Reply:
The interesting thing about this is that this is a React Navigation thing, and needs to be like this. Technically `undefined` represents the params passed to it (even though we don't actually use it sorry not sorry).